cypress/e2e/firstTest.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('First Test', () => {
+
+    it('Check the page', () => {
+        cy.visit('http://localhost:4200/pages/layout/stepper');
```

Comment:
If you have assigned base url in config file, you dont need to put  full url here.
```suggestion
        cy.visit('pages/layout/stepper');
```